src/pages/my/community.vue

```diff
@@ -254,52 +505,118 @@ export default {
         .catch(err => {
           this.error = err;
           this.is_submitting = false;
+          for (let language in err.response.data) {
+            const response = JSON.parse(err.response.data[language].replace(/'/g, "\""));
+            for (let item in response) {
+              const feedback = response[item];
+              let key = item + '_' + language.toLowerCase();
+              this.errors[key] = feedback;
+            }
+          }
         });
     },
+    openTab(evt, tabName) {
+      // Declare all variables
+      var i, tabcontent, tablinks;
+
+      // Get all elements with class="tabcontent" and hide them
+      tabcontent = document.getElementsByClassName("tabcontent");
+      for (i = 0; i < tabcontent.length; i++) {
+        tabcontent[i].style.display = "none";
+      }
+
+      // Get all elements with class="tablinks" and remove the class "active"
+      tablinks = document.getElementsByClassName("tablinks");
+      for (i = 0; i < tablinks.length; i++) {
+        tablinks[i].className = tablinks[i].className.replace(" active", "");
+      }
+
+      // Show the current tab, and add an "active" class to the button that opened the tab
+      document.getElementById(tabName).style.display = "block";
+      evt.currentTarget.className += " active";
+    },
     /**
      * Generate the FormData
      * @returns {FormData}
      */
     normalizeFormData() {
       let data = new FormData();
+      let data_nl = {language_code: 'NL'};
+      let data_en = {language_code: 'EN'};
 
+      // TODO lodash foreach
       for (let item in this.formData) {
         const el = this.formData[item];
         if (el) {
+          let value = null;
           if (Array.isArray(el)) {
-            data.append(item, JSON.stringify(el));
+            value = JSON.stringify(el);
           } else {
             let ElValue = el ? el : null;
-            data.append(item, ElValue);
+            value = ElValue;
+          }
+          if (!item.startsWith('logo') && (!item.startsWith('featured'))){
+            if (item.endsWith('_nl')) {
+              data_nl[item.slice(0, -3)] = value;
+            } else if (item.endsWith('_en')) {
+              data_en[item.slice(0, -3)] = value;
+            }
           }
+          data.append(item, value);
         }
+        // if the value is empty, send it to the backend (so the backend can reject the post)
+        else {
+          if (item.endsWith('_nl')) {
+              data_nl[item.slice(0, -3)] = "";
+          } else if (item.endsWith('_en')) {
+              data_en[item.slice(0, -3)] = "";
+          }
+        }
+      }
+      let deleted_logos = [];
+      data.set('logo_nl', '');
+      if (this.logo_nl_added) {
+        let logo = this.$refs['file-logo_nl'].$el.querySelector('input[type="file"]').files[0];
```

Comment:
Not super happy about querySelector. Could be replaced with $refs, but never mind for now.